src/main/resources/public/src/views/Validation.vue

```diff
@@ -1,43 +1,68 @@
 <template>
     <div class="container">
         <h2>Access validation</h2>
-        <data-access-validation></data-access-validation>
-        <router-link to="/">
-            <input class="validation-buttons" type="button" @click="validAccess" value="Valid" v-bind:disabled="disabled">
-        </router-link>
-        <router-link to="/">
-            <input class="validation-buttons" type="button" @click="refuseAccess" value="Refuse">
-        </router-link>
+        <p>{{ dataRequest.model }}</p>
+        <div class="form-check">
```

Comment:
Why didn't you keep the data-access-validation component ? Wasn't it good ?

And, this is totally a personnal point of view but, a disabled input with dataRequest.model as value wouldn't have a better look ?